components/form_inputs/autocompletes/AutocompleteStyle.vue

```diff
@@ -0,0 +1,191 @@
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    dense
+                    rounded
+                    filled
+                    label="Filled rounded"
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    dense
+                    filled
+                    outlined
+                    label="Filled outlined"
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Solo rounded"
+                    dense
+                    rounded
+                    solo
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Solo-inverted outlined"
+                    dense
+                    outlined
+                    solo-inverted
+            ></v-autocomplete>
+          </v-col>
+        </v-row>
+      </v-container>
+    </v-card>
+
+    <h3>Dark</h3>
+    <v-card dark>
+      <v-container>
+        <v-row
+                align="center"
+        >
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Shaped outlined"
+                    dense
+                    shaped
+                    outlined
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    dense
+                    rounded
+                    filled
+                    label="Filled rounded"
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    dense
+                    filled
+                    outlined
+                    label="Filled outlined"
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Solo rounded"
+                    dense
+                    rounded
+                    solo
+            ></v-autocomplete>
+          </v-col>
+
+          <v-col cols="12">
+            <v-autocomplete
+                    v-model="values"
+                    :items="items"
+                    label="Solo-inverted outlined"
+                    dense
+                    outlined
+                    solo-inverted
+            ></v-autocomplete>

```

Comment:
In matrix only 'dark', 'rounded' and 'solo-inverted' are mentioned.
Could you, please, clarify for what purpose you add theses extra 5 autocompletes?
It may confuse a little bit :)